app/controllers/refunds_controller.rb

```diff
@@ -44,6 +44,15 @@ def create
 
   private
 
+  def refunder
```

Comment:
Can we not determine whether Govpay or Worldpay based on `payment[:payment_type]` ?